roles/cuda_toolkit_install/tasks/verify.yml

```diff
@@ -0,0 +1,44 @@
+# Manually Verify Supported Version of Linux and Verify Correct Kernel Headers and
+# Development Packages at https://docs.nvidia.com/cuda/cuda-installation-guide-linux/index.html
+
+# Verify gcc
+- name: Verify gcc
+  ansible.builtin.shell:
+    "command -v gcc >/dev/null 2>&1"
+  register: gcc_command
+  changed_when: false
+  ignore_errors: true
+
+- name: Install gcc
```

Comment:
Do not install `gcc` just for a check.

Verify if `gcc` is installed. Print error and exit if not.